src/ts/components/settingspanel.ts

```diff
@@ -97,9 +97,11 @@ export class SettingsPanel extends Container<SettingsPanelConfig> {
         this.hide();
         this.hideHoveredSelectBoxes();
       });
-      this.getDomElement().on('mouseenter', () => {
-        // On mouse enter clear the timeout
-        this.hideTimeout.clear();
+      this.getDomElement().on('mouseenter mousemove', () => {
```

Comment:
I had to add the `mousemove` event here as well because the `focusout` event triggers between `SettingsPanelPage` transitions causing the settings panel to hide after selecting anything.